gulpfile.js

```diff
@@ -171,77 +205,44 @@ gulp.task('coverage:instrument', function () {
         .pipe(istanbul.hookRequire());
 });
 
-gulp.task('coverage:report', function (done) {
+gulp.task("coverage:report", function (done) {
     return gulp.src(
         ["src/**/*.js", "!test/**"],
         { read: false }
     )
         .pipe(istanbul.writeReports({
-            reporters: ['json', 'text-summary']
+            reporters: ["json", "text-summary"]
         }));
 });
 
-gulp.task('coverage:remap', function () {
-    return gulp.src('coverage/coverage-final.json')
+gulp.task("coverage:remap", function () {
+    return gulp.src("coverage/coverage-final.json")
         .pipe(remapIstanbul({
             reports: {
-                'json': 'coverage/coverage.json',
-                'html': 'coverage/html-report'
+                "json": "coverage/coverage.json",
+                "html": "coverage/html-report"
             }
         }));
 });
 
-gulp.task("test:coverage", function (done) {
-    runSequence("quick-build", 'coverage:instrument',
-        "test-no-build", 'coverage:report', 'coverage:remap', done);
-});
-
 gulp.task("test-no-build", test);
 
-gulp.task("check-imports", function (cb) {
-    return tsProject.src()
-        .pipe(imports());
-});
-
-gulp.task("check-copyright", function (cb) {
-    return gulp.src([
-        "**/*.ts",
-        "**/*.js",
-        "!**/*.d.ts",
-        "!coverage/**",
-        "!node_modules/**",
-        "!test/**/*.js",
-        "!SampleApplication/**",
-        "!test/resources/sampleReactNative022Project/**/*.js"
-    ])
-        .pipe(copyright());
-});
-
-gulp.task("watch-build-test", ["build", "build-test"], function () {
-    return gulp.watch(sources, ["build", "build-test"]);
-});
+gulp.task("test:coverage", gulp.series("quick-build", "coverage:instrument",
+"test-no-build", "coverage:report", "coverage:remap", function (done) {
```

Comment:
The last callback seems to be unnessesary.

Reply:
Fixed.